src/theme/bootstrap-override/bootstrap-italia/_header.scss

```diff
@@ -15,5 +15,11 @@
     .it-nav-wrapper .it-header-navbar-wrapper {
       margin-top: -20px;
     }
+
+    .navbar-toggler {
+      svg.fa-icon {
+        fill: $white;
+      }
+    }
```

Comment:
Temo non vada bene, abbiamo diversi temi con sfondo bianco nell'header e la gestione era già delicata prima. Vediamo magari assieme per vedere se ho capito bene e in caso se troviamo un'alternativa.

Reply:
Per i temi con header bianco, c'è un file apposito se non sbaglio, che all'interno ha questo codice (fixato in un'altra PR)
```
 @media (max-width: map-get($map: $grid-breakpoints, $key: 'lg') - 1px) {
      button.custom-navbar-toggler svg.icon {
        fill: color-contrast($primary);
      }
    }
```

Che dovrebbe risolvere il problema
Volendo, al posto di $white, si può impostare un color-contrast con la variabile dello sfondo dell'header?